patches/minecraft/net/minecraft/world/gen/DimensionSettings.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/world/gen/DimensionSettings.java
++++ b/net/minecraft/world/gen/DimensionSettings.java
+@@ -104,7 +104,7 @@
+    }
+ 
+    private static DimensionSettings func_242741_a(DimensionStructuresSettings p_242741_0_, BlockState p_242741_1_, BlockState p_242741_2_, ResourceLocation p_242741_3_) {
+-      Map<Structure<?>, StructureSeparationSettings> map = Maps.newHashMap(DimensionStructuresSettings.field_236191_b_);
++      Map<Structure<?>, StructureSeparationSettings> map = net.minecraftforge.common.ForgeHooks.getAllStructureSeparationSettings();
```

Comment:
A problem that might occur with this is that the method is called during `clinit`, and if a mod causes classloading of this before things are registered it will break